ndcli/dimcli/__init__.py

```diff
@@ -2017,9 +2018,7 @@ def rename_zoneprofile(self, args):
         '''
         Renames a zone profile.
         '''
-        options = OptionDict(profile=True)
```

Comment:
The `profile` option is still required in the request to DIM, breaks: https://github.com/1and1/dim/blob/2c8ca6e3ea8d4082dce1f5f800de4a7aff690387/dim-testsuite/t/history-zone.t#L13

Reply:
Oh, thank you!
There are tests, but I thought these were for a missing feature :(